cmd/image/qcow2ova/prep/templates.go

```diff
@@ -66,10 +74,19 @@ defaults {
 }
 EOF
 sed -i 's/GRUB_TIMEOUT=.*$/GRUB_TIMEOUT=60/g' /etc/default/grub
+{{if eq .Dist "fedora"}}
+sed -i 's/GRUB_CMDLINE_LINUX_DEFAULT=.*$/GRUB_CMDLINE_LINUX="console=tty0 console=hvc0,115200n8  biosdevname=0  crashkernel=auto rd.shell rd.debug rd.driver.pre=dm_multipath log_buf_len=1M "/g' /etc/default/grub
+{{else}}
```

Comment:
can we use something like `sed -i 's/^\(GRUB_CMDLINE_LINUX_DEFAULT\|GRUB_CMDLINE_LINUX\)=.*$/GRUB_CMDLINE_LINUX="console=tty0 console=hvc0,115200n8 biosdevname=0 crashkernel=auto rd.shell rd.debug rd.driver.pre=dm_multipath log_buf_len=1M "/g' /etc/default/grub`

Reply:
ok, will update